internal/pkg/cli/app_package.go

```diff
@@ -277,6 +325,19 @@ func (o *packageAppOpts) setFileWriters() error {
 		return fmt.Errorf("create file %s: %w", paramsPath, err)
 	}
 	o.paramsWriter = paramsFile
+
+	return nil
+}
+
+func (o *packageAppOpts) setAddonsFileWriter() error {
```

Comment:
If `OutputDir==""` then I think we still try to create the file right?

Can we do something like:
```
if o.OutputDir == "" {
     o.addonsWriter = ioutil.Discard
     return nil
}
```